lib/active_job/uniqueness/sidekiq_patch.rb

```diff
@@ -91,6 +91,10 @@ def delete_by_value(name, value)
 
 # Global death handlers are introduced in Sidekiq 5.1
 # https://github.com/mperham/sidekiq/blob/e7acb124fbeb0bece0a7c3d657c39a9cc18d72c6/Changes.md#510
-if sidekiq_version >= Gem::Version.new('5.1')
+if sidekiq_version >= Gem::Version.new('7.0')
+  Sidekiq.configure_server do |config|
+    config.death_handlers << ->(job, ex) { ActiveJob::Uniqueness.unlock_sidekiq_job!(job) }
```

Comment:
```suggestion
    config.death_handlers << ->(job, _ex) { ActiveJob::Uniqueness.unlock_sidekiq_job!(job) }
```
I guess Rubocop will complain about the variable name if it is not bein used in the block

Reply:
Yep, It [does complain](https://github.com/veeqo/activejob-uniqueness/actions/runs/3360350642/jobs/5604585189#step:5:22)

Reply:
Whoops - don't know how I stuffed that considering it's the same as the line below it.